central/tlsconfig/tlsconfig_test.go

```diff
@@ -257,3 +258,28 @@ func writeKey(dir string, privateKey *ecdsa.PrivateKey) error {
 	}
 	return pem.Encode(keyFile, &pem.Block{Type: "EC PRIVATE KEY", Bytes: privateKeyBytes})
 }
+
+func Test_isValidAdditionalCAFileName(t *testing.T) {
```

Comment:
```suggestion
func TestIsValidAdditionalCAFileName(t *testing.T) {
```

Reply:
Could you clarify your stance on https://github.com/stackrox/stackrox/pull/5591#discussion_r1165197016